content/ref/python/watch.md

```diff
@@ -22,12 +22,7 @@ extended to support arbitrary machine learning models in the future.
 
 | Args |  |
 | :--- | :--- |
-|  `models` | A single model or a sequence of models to be monitored. | 
-|`criterion `| The loss function being optimized (optional). |
-|`log `| Specifies whether to log `gradients`, `parameters`, or `all`. Set to `None` to disable logging. (default="gradients") |
-|`log_freq `|: How frequently to log gradients and parameters, expressed in batches. (default=1000)| 
-|`idx `| Index used when tracking multiple models with `wandb.watch`. (default=None)| 
-|`log_graph `| Whether to log the model's computational graph. (default=False) |
+|  models (Union[torch.nn.Module, Sequence[torch.nn.Module]]): A single model or a sequence of models to be monitored. criterion (Optional[torch.F]): The loss function being optimized (optional). log (Optional[Literal["gradients", "parameters", "all"]]): Specifies whether to log "gradients", "parameters", or "all". Set to None to disable logging. (default="gradients") log_freq (int): Frequency (in batches) to log gradients and parameters. (default=1000) idx (Optional[int]): Index used when tracking multiple models with `wandb.watch`. (default=None) log_graph (bool): Whether to log the model's computational graph. (default=False) |
```

Comment:
Nit: These would all be fixed by using code font and removing the extra whitespace, in the source. Perhaps we need to socialize the sort of Markdown we would like to see used.